### ISSUE TITLE
BERT lr scheduler 

### ISSUE THREAD

**Issue by eric-haibin-lin** [dmlc/gluon-nlp#1173]
We can move the lr scheduling logic https://github.com/dmlc/gluon-nlp/blob/master/scripts/bert/finetune_classifier.py#L565-L571 for BERT to a LRScheduler API implementing the `mxnet.lr_scheduler.LRScheduler` API

**Comment by sxjscience** [comment]
@eric-haibin-lin Have changed to use the LRScheduler in the numpy version: https://github.com/sxjscience/numpy_nlp/blob/315274087a8b6e6bc33f883062486cb52c7552c1/src/numpy_nlp/lr_scheduler.py#L5
